docs/operations/metrics.md

```diff
@@ -413,7 +413,7 @@ These metrics are emitted by the Druid Coordinator in every run of the correspon
 |`tier/total/capacity`|Total capacity in bytes available in each tier.|`tier`|Varies|
 |`compact/task/count`|Number of tasks issued in the auto compaction run.| |Varies|
 |`compactTask/maxSlot/count`|Maximum number of task slots available for auto compaction tasks in the auto compaction run.| |Varies|
-|`compactTask/availableSlot/count`|Number of available task slots that can be used for auto compaction tasks in the auto compaction run. This is the max number of task slots minus any currently running compaction tasks.| |Varies|
+|`compactTask/availableSlot/count`|Number of task slots available for auto compaction tasks during an auto compaction run. This value represents the total number of task slots allocated for compaction minus the estimated number of currently running compaction tasks. The estimated number of tasks is capped by `maxNumConcurrentSubTasks`, which serves as an upper bound to prevent over-allocation of compaction tasks.| |Varies|
```

Comment:
Slight change of wording to avoid ambiguity. Let me know if it makes sense.

```suggestion
|`compactTask/availableSlot/count`|Number of currently vacant task slots that are available for auto-compaction tasks. This value is computed as the difference between the total number of task slots allocated for auto-compaction minus the estimated number of task slots currently occupied by running compaction tasks. The number of sub-tasks of each compaction task is estimated to be `maxNumConcurrentSubTasks`.| |Varies|
```

Reply:
Yes, it does, thanks. Just one small tweak, I left it as auto compaction instead of auto-compaction to keep it consistent with other metrics in this block.